distributions/demo-google-deployment/transfer/build.gradle

```diff
@@ -26,29 +45,69 @@ repositories {
 
 dependencies {
     compile project(':portability-transfer')
-
-    compile project(':extensions:data-transfer:portability-data-transfer-flickr')
+    compile project(':extensions:config:portability-config-yaml')
+    // TODO: depend on these based on list in flag values.
     compile project(':extensions:data-transfer:portability-data-transfer-google')
+    compile project(':extensions:data-transfer:portability-data-transfer-instagram')
     compile project(':extensions:data-transfer:portability-data-transfer-microsoft')
-    compile project(':extensions:cloud:portability-cloud-google')
-    compile project(':extensions:config:portability-config-yaml')
+    compile project(':extensions:data-transfer:portability-data-transfer-flickr')
+    compile project(':extensions:data-transfer:portability-data-transfer-rememberthemilk')
 }
 
 addCloudExtensionDependency(project)
 
 mainClassName = 'org.dataportabilityproject.transfer.WorkerMain'
 
-copyResources()
-
 shadowJar {
     mergeServiceFiles()
     exclude '**/pom.properties'
     exclude '**/pom.xml'
-}
-
-def copyResources() {
     copy {
         from '../resources/'
         into 'src/main/resources'
     }
 }
+
+// Note: This service acct creds file should be used for local testing only. In production, we use
```

Comment:
Do we need to make this flag driven?  Or, ideally, can we have a local and prod deploy script that overrides this?

Reply:
Yeah, I was going to come back to this once I start looking into the prod setup. I have a note at the top about the next step being to support non-local environments, I will come back to that in the next PR